### ISSUE TITLE
Acessing Rider Preferences in Unity Editor 2018.20f2 Results in Blank Window

### ISSUE THREAD

**Issue by jwvanderbeck** [JetBrains/resharper-unity#665]
Unity 2018.2.0f2
Rider 2018.2 EAP (Build RD-182.3894.140)
If I try to access the Rider preferences inside the Unity Editor I just get a blank window and continual errors in the console.

```
InvalidOperationException: Sequence contains no elements
GUI Error: You are pushing more GUIClips than you are popping.  Make sure they are balanced
```

**Comment by citizenmatt** [comment]
Is there a call stack for the `InvalidOperationException`? I think that's the one causing the problem. Are you on Windows?

**Comment by jwvanderbeck** [comment]
This is on a Windows machine but one not connected to a network so I would
gave to type the whole thing out.

Let me see if I can reproduce it on my home PC this weekend.

- John Vanderbeck
- http://www.johnvanderbeck.com

**Comment by citizenmatt** [comment]
Thanks. If it helps, I think the exception might be thrown by [this call to `Enumerable.Aggregate`](https://github.com/JetBrains/resharper-unity/blob/182/unity/EditorPlugin/PluginSettings.cs#L252), which implies the [`GetInstalledDotNetFrameworks` method is returning an empty array](https://github.com/JetBrains/resharper-unity/blob/182/unity/EditorPlugin/PluginSettings.cs#L37). Perhaps this will help diagnose what's going on.

**Comment by jwvanderbeck** [comment]
Ah very possible.  This is a new machine I just got, and systems normally
puts Visual Studio on there, which most likely would install the DotNET
packages, but I told them not to bother because I only want to use Rider
:)  I did notice Rider itself telling me 4.7 was missing, and I was going
into the preferences to change it to 4.,6 because that is what I normally
use.  Chances are 4.6 isn't on there either like you said.  So you most
likely nailed it on the head.

I'll fix up that code to not barf out if no frameworks are installed and
see what that does.

- John Vanderbeck
- http://www.johnvanderbeck.com
